### PR TITLE
Schedule deletion of users

### PR DESCRIPTION
**Pull request by acasajus**
users to be scheduled to be deleted cannot:
 - receive or send emails
 - login
 
 For now the cron is disabled in case this gets deployed to prod before extensive tests

### CODE REVIEW

File: cron.py [hunk 1]
```diff
@@ -1106,6 +1106,18 @@ def notify_hibp():
...
 
+def clear_users_scheduled_to_be_deleted():
+    users = User.filter(
+        and_(User.delete_on.isnot(None), User.delete_on < arrow.now())
+    ).all()
+    for user in users:
+        LOG.i(
+            f"Scheduled deletion of user {user} with scheduled delete on {user.delete_on}"
+        )
+        user.delete(user.id)
```
Comment (nguyenkims) [inline cron.py:1117]: this should rather be `Session.delete(user)` right?

**Review by nguyenkims — COMMENTED** [review]
LGTM, just 1 comment.

File: cron.py (hunk 1 above)
Comment (acasajus) [reply cron.py:1117]: It's the same. I did it the same way as the job runner does https://github.com/simple-login/app/blob/master/job_runner.py#L222

Comment (nguyenkims) [reply cron.py:1117]: In this one it's `User.delete` (note that uppercase U)

Comment (acasajus) [reply cron.py:1117]: good one! Then how the ** do the test work :thinking: ?

Comment (acasajus) [reply cron.py]: ah, bc it's a static method and the user object calls it regardless.. Changed it.

Comment (nguyenkims) [reply cron.py]: Oh that's true.

**Pull request merged**